src/configs/novaposhta_api.ts

```diff
@@ -0,0 +1,9 @@
+const novaposhta_api = (modelName: string, calledMethod: string, methodProps: any) => {
```

Comment:
better rename to `getNovaposhtaPayload`
but not sure about do we need this function at all, all wat it does - add apiKey